stages/org.osbuild.coreos.osmet

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python3
+
+import glob
+import hashlib
+import os
+import subprocess
+import shutil
+import sys
+
+import osbuild.api
+
+
+def sha256sum_file(path):
+    """
+    Calculates the sha256 sum from a path.
+
+    :param path: The full path to the file
+    :type: path: str
+    :returns: The calculated sha256 sum
+    :type: str
+    """
+    h = hashlib.sha256()
+    with open(path, 'rb', buffering=0) as f:
+        for b in iter(lambda: f.read(128 * 1024), b''):
+            h.update(b)
+    return h.hexdigest()
+
+
+def parse_os_release(deployed_tree):
+    """
+    Get the OS release.
+
+    :param deployed_tree: The full path to the tree
+    :type: path: str
+    :returns: The OS release name
+    :type: str
+    """
+    r = dict()
+    with open(os.path.join(deployed_tree, 'etc/os-release')) as f:
+        for line in f:
+            k, v = line.split('=', 1)
+            r[k] = v.strip(" \n\t'\"")
+    return r
+
+def bwrap(usr, etc, cmd):
```

Comment:
Can we move this to a utility/separate file and then break down this stage into an osmet stage and a miniso stage that both reuse the bwrap function?

Reply:
see also https://github.com/ravanelli/osbuild/pull/1#discussion_r1781628846